DigitalLearningSolutions.Web.Tests/Controllers/Register/RegisterAtNewCentreControllerTests.cs

```diff
@@ -135,11 +138,37 @@ public void PersonalInformationPost_with_invalid_emails_fails_validation()
             var result = controller.PersonalInformation(model);
 
             // Then
-            A.CallTo(() => userService.NewEmailAddressIsValid(model.CentreSpecificEmail!, ControllerContextHelper.UserId))
+            A.CallTo(
+                    () => userService.NewEmailAddressIsValid(model.CentreSpecificEmail!, ControllerContextHelper.UserId)
+                )
                 .MustHaveHappened();
             result.Should().BeViewResult().WithDefaultViewName();
         }
 
+        [Test]
+        public void PersonalInformationPost_with_null_centre_email_skips_email_validation()

```

Comment:
This isn't testing anything new that I've implemented, just looked like a good test to have that didn't already exist for some reason.